src/scripts/exclusion_reasons_enum_updater.py

```diff
@@ -0,0 +1,61 @@
+"""Updates exclusion reasons enum: src/schema/mondo.yaml
+
+Resources:
+- Source CSV: https://docs.google.com/spreadsheets/d/18V-_j4b22LzCytxFEM3pBEl6-_ODJY06gpUwH0F6JVY/edit#gid=1644570180
+
+Instructions:
+1. Save the 'exclusion code and definitions' tab from the above GoogleSheet as `src/scripts/exclusion_reasons.csv`
+"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict
+
+import pandas as pd
+import yaml
+
+
+GSHEETS_URL = 'https://docs.google.com/spreadsheets/d/18V-_j4b22LzCytxFEM3pBEl6-_ODJY06gpUwH0F6JVY/edit#gid=1644570180'
+PROJECT_DIR = Path(os.path.dirname(__file__)).parent.parent
+DEFAULTS = {
```

Comment:
I know defaults discouraged, but this is not `mondo-ingest`; interested more in development speed than convention at this moment.

Reply:
Just keep in mind that this should be changed in production if we want to keep these scripts transparent.

Reply:
Hmm, I understand changing them in production in order to discourage simply running the file without knowing what it does or where the inputs are; but RE: transparency, not sure what you mean specifically.

Perhaps we should leave this as it is now until we have an automated way to get stuff from GoogleSheets to CSV programmatically.

There's a way to do that in Python, but would be nice if there was an easy way to do this from the shell. Maybe there's a way. The Python way I know still involves a lot of code, and multiple JSON files which have the API key in them; kinda messy.